src/test/java/training/article/update.feature

```diff
@@ -0,0 +1,32 @@
+Feature: update an article
+
+  Background:
+    * url url
+    * def username = username
+    * def getArticles = call read('getArticle.feature@getAllArticles')
+    * def randomArticle = read('../helpers/Random.js')
+    * def slug = typeof passedSlug == 'undefined' ? randomArticle(getArticles.response.articles).slug : passedSlug
```

Comment:
remember, if you passed a slug, you don't need to `call` the scenario getAllArticles